client/src/components/Banner.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react'
+import { useBanner } from 'hooks/useBanner'
+import { Box, Paragraph } from 'grommet'
+import { Link } from 'components/Link'
+import { Icon } from 'components/Icon'
+import styled, { css } from 'styled-components'
+
+export const Banner = ({
+  bannerLabel,
+  bgColor = 'brand',
+  ctaLabel,
+  ctaLink,
+  fontColor = 'white',
+  htmlIcon,
+  iconName,
+  iconSize,
+  children
+}) => {
+  const { showing, setShowing, onBannerClose } = useBanner()
+  const BannerBox = styled(Box)`
+    ${({ theme }) => css`
+      background-color: ${theme.global.colors[bgColor] ||
+      theme.global.colors.gradient[bgColor]};
+      > div {
+        > p {
+          > a {
+            text-decoration: underline;
+          }
+        }
+      }
+    `}
+  `
+  useEffect(() => {
+    setShowing(true)
```

Comment:
instead of hard coding `true` can you make this a prop, add something like `open = true`